app/serializers/did_not_vote_serializer.rb

```diff
@@ -0,0 +1,15 @@
+class DidNotVoteSerializer < ActiveModel::Serializer
+  embed :ids, include: true
+  attributes :id, :created_at
+
+  has_one :user, serializer: UserSerializer, root: 'users'
+  has_one :proposal, serializer: MotionSerializer, root: 'proposals'
```

Comment:
man, this thing again, heh.

I'd be curious if `has_one :motion, key: :proposal, serializer: MotionSerializer, root: 'proposals'` would work.